packages/manager/modules/sms/src/sms/batches/details/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,7 @@
+{
+  "sms_batches_details_title": "Détail de la campagne",
+  "sms_batches_details_name": "Campagne: {{name}}",
+  "sms_batches_details_message": "Message délivré",
```

Comment:
Don't find where this translation is used.

Reply:
thanks for feedback, i've not found this one on any part of code. i've removed it on last commit :)